Apps/US/DynamicsGPMigrationVendor1099/test/app.json

```diff
@@ -0,0 +1,52 @@
+{
```

Comment:
Same comment here - Do we need this app? Could we join it to the existing app?

Reply:
See previous response on this topic